.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+### Cookbook version
```

Comment:
This can go, the community health files will automatically be used instead: https://github.com/chef-cookbooks/.github

Reply:
corrected